src/common/components/table/Table.tsx

```diff
@@ -15,7 +15,7 @@ import {
   getSortedRowModel,
   useReactTable,
 } from '@tanstack/react-table';
-import React, { useEffect, useMemo, useState } from 'react';
+import React, { type JSX, useEffect, useMemo, useState } from 'react';
```

Comment:
why are we adding this import?

Reply:
it's used in this component, this is mapped automatically for react19 using codemods